settings/default/search.lua

```diff
@@ -11,8 +11,8 @@ xi.settings = xi.settings or {}
 
 xi.settings.search =
 {
-    -- Omit items with no buy history from auction house results
-    OMIT_NO_HISTORY = false,
+    -- Omit items with no listing history from auction house results
+    OMIT_NO_HISTORY = true,
```

Comment:
despite being technically retail, we probably want to default it off anyway if only because ppl who didn't read/notice it will then think its bugged

Reply:
I agree people probably expect the false behavior but I'm conflicted because I think settings should default to retail-like. Existing servers won't notice the difference if they have this set already in their copy of the settings file and it's one of only a few settings for the search server so would be hard to miss.

Reply:
Normally we want all default retail. I think this and thinking show all was better than retail was the main reasoning behind defaulting off in the PR it came in on. I'm ok with whatever other devs prefer here, but thought it worth mentioning since we had it off in here already.

Reply:
> one of only a few settings for the search server so would be hard to miss.

You would think, but people often don't notice when new settings land in the default files to add them to their edited files and wind up with whatever the default setting behavior is (or an error). Even catches me now and then after I pull on my testbed not realizing how long its been. One of the few drawbacks of making it so nobody is accidently committing their settings anymore :)

Reply:
I wanted to squash that style commit so I changed the default setting back to false since I was already rebasing. If you guys decide you want it the other way just let me know and I'll change it back np.